lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts

```diff
@@ -199,13 +199,13 @@ export class DropdownCloudWidgetComponent extends WidgetComponent implements OnI
     }
 
     private updateFormControlState(): void {
-        const isFieldRequired = this.isRequired();
+        this.dropdownControl.setValidators([]);
 
-        this.dropdownControl.setValidators(isFieldRequired && this.field?.isVisible ? [Validators.required] : []);
-
-        const addSelectDefaultOptionValidator = isFieldRequired && this.field.hasEmptyValue;
-        if (addSelectDefaultOptionValidator) {
-            this.dropdownControl.addValidators([defaultValueValidator(this.field)]);
+        if (this.isRequired() && this.field?.isVisible) {
+            this.dropdownControl.addValidators([Validators.required]);
+            if (this.field.hasEmptyValue) {
+                this.dropdownControl.addValidators([defaultValueValidator(this.field)]);
+            }
```

Comment:
if the field is REQUIRED and VISIBLE - add the required validator
if the field is REQUIRED and VISIBLE and CONTAINS THE DEFAULT OPTION - add the defaultValueValidator

Reply:
maybe we could `disable` the control if is hidden?